mkdocs.yml

```diff
@@ -0,0 +1,11 @@
+site_name: aws_resource_validator
+theme:
+  name: material
+
+nav:
+  - Home: index.md
+  - Usage: usage.md
+  - API Reference: api.md
+
+plugins:
+  - search
```

Comment:
what does this plugin mean?

Reply:
This plugin enables a full-text search feature on the documentation site, allowing users to easily search for keywords or phrases within your site's content.